example/data-migration-ecs/terraform.tfvars.example

```diff
@@ -0,0 +1,8 @@
+permissions_boundary_arn = "arn:aws:iam::12345:policy/NGAPShRoleBoundary"
+prefix = "PREFIX"
+subnet_ids = ["subnet-12345"]
+vpc_id = "vpc-12345"
+
+data_migration2_function_arn = "arn:aws:lambda:us-east-1:<12345>:function:PREFIX-data-migration2"
+log2elasticsearch_lambda_function_arn = "arn:aws:lambda:us-east-1:<12345>:function:PREFIX-log2elasticsearch"
+ecs_cluster_arn = "arn:aws:ecs:us-east-1:<12345>:cluster/PREFIX-tf-CumulusECSCluster"
```

Comment:
Will probably remove. See above.

Reply:
I'm not sure you actually *do* want to remove this - see my comment re: structure.   

I'm guessing we want a configurable module that then is called by an example project that does the remote state lookups.